Mathlib/Data/List/Basic.lean

```diff
@@ -1536,25 +1537,28 @@ theorem scanr_cons (f : α → β → β) (b : β) (a : α) (l : List α) :
   | cons hd tl ih => simp only [foldr, ih]
 
 section FoldlEqFoldr
```

Comment:
I thought mathlib style is to leave a blank line after sections? Not a blocking concern, in any case.
```suggestion
section FoldlEqFoldr

```